src/ui/React/AlertManager.tsx

```diff
@@ -46,9 +46,11 @@ export function AlertManager({ hidden }: { hidden: boolean }): React.ReactElemen
 
   const alertMessage = alerts[0]?.text || "No alert to show";
 
-  function getMessageHash(text: string | JSX.Element): string {
-    if (typeof text === "string") return sha256(text);
-    return sha256(JSON.stringify(text.props));
+  async function getMessageHash(text: string | JSX.Element): Promise<string> {
```

Comment:
This seems needlessly async, and it has potential race condition implications.

Reply:
I forgot removing them.